public/js/navigator-js/integration/ReactRecipe.js

```diff
@@ -0,0 +1,86 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+var ReactRecipe = {
+  _type: 'REACT',
+
+  // Returns a fake proxy element due to the fact that
+  // we do not use these transition functions within
+  // our react components
+
+  getViewInstance: function getViewInstance() {
+    return {
+      navigatorBehaviors: ['IHasStateTransition'],
+      transitionIn: function(cb) { cb() },
```

Comment:
I am not sure I get this. 
The concept behind transitionIn/Out is to allow for timed transition - 
transitionIn(cb) {
// view does some animation or wait for some data or anything timed....
// then transitionIn -> callback ===> trigger transitionOut for previous view instance 
cb();
}

By making these proxies are we fully losing this feature?